packages/next-plugin-treat/package.json

```diff
@@ -0,0 +1,22 @@
+{
+  "name": "next-plugin-treat",
+  "version": "1.0.0",
+  "description": "Next.js plugin for treat",
+  "keywords": [
+    "nextjs",
+    "nextjs-plugin",
+    "treat"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/seek-oss/treat.git",
+    "directory": "packages/next-plugin-treat"
+  },
+  "license": "MIT",
+  "dependencies": {
+    "mini-css-extract-plugin": "^0.9.0"
```

Comment:
Please mark this as a peer

Reply:
Thank you for all the valuable feedback, I've fixed the 3 issues.

Reply:
I'm not sure this is right. Consumers aren't installing this package themselves—it's actually a dependency of Next—so they'll get peer dependency warnings, right? I'm concerned that this isn't the right way for a Next plugin to output CSS. Tried finding other Next plugins that import `mini-css-extract-plugin` directly and can't find any examples.

Reply:
I’m not quite familiar with the architecture of Next. About the peer dependencies, I had the same concerns and could make it opt-in with a `peerDependenciesMeta` field. Alternative methods for encapsulating the `mini-css-extract-plugin` would be great.

There’s an RFC discussing the form of upcoming Next plugins at https://github.com/zeit/next.js/issues/9133. Unfortunately, it seems to lack webpack config modification support intentionally, due to concerns about performance.

Plugins are not widely used within the Next ecosystem. Current solutions aren’t highly documented, which explains the low amount of plugins available.